Marlin/src/module/planner.h

```diff
@@ -268,6 +268,8 @@ typedef struct PlannerBlock {
              final_adv_steps;               // Advance steps for exit speed pressure
   #endif
 
+  #define MINIMAL_STEP_RATE _MAX((STEPPER_TIMER_RATE / HAL_TIMER_TYPE_MAX), 1UL)   // min. steps/s. To prevent timer overflow, slowest is 1 step/s
```

Comment:
```C++
  #ifdef CPU_32_BIT
    // min. steps/s. To prevent timer overflow, slowest is 1 step/s
    #define MINIMAL_STEP_RATE _MAX((STEPPER_TIMER_RATE / HAL_TIMER_TYPE_MAX), 1UL) 
  #else
    #define MINIMAL_STEP_RATE (F_CPU) / 500000U // i.e., 32 or 40
  #endif
```

And use in `Stepper::calc_timer_interval` so the two are kept in sync.